Merge and rewrite download with other chapter (#1793)
Deltares/Ribasim: docs/install.qmd

title: "Installation"
---

> In this document, we describe how to install the Ribasim software package.

Reviewer: ```suggestion
In this document, we describe how to install the different components of Ribasim.
First the components are introduced, then installation instructions per component follow.
```

> In this document, we describe how to install the Ribasim software package.

Reviewer: ```suggestion
In this document, we describe how to install the different components of Ribasim.
First the components and their relation are introduced, then installation instructions per component follow.
```

# Components {#sec-components}

> The figure below illustrates the relation between the various components of the Ribasim software package.

Reviewer: ```suggestion
The figure below illustrates the relation between the various components of Ribasim.
```

> # Download {#sec-download}
> 

Reviewer: ```suggestion
```

Let's remove this section entirely since it's no longer just about downloading.

> There are three main components of the Ribasim software package.
> They are the Ribasim core (written in Julia language), the Ribasim Python package and the Ribasim QGIS plugin.
> The Ribasim core is the computation kernel of the Ribasim software package and is used to run simulations.
> The Ribasim Python package is used to build Ribasim models programmatically with Python script.
> The Ribasim QGIS plugin is used to construct or view a model in GUI.
> Users can choose to use Ribasim Python or QGIS plugin or a combination of them to build a Ribasim model.
> And then use the Ribasim core to run the simulation.
> 

Reviewer: ```suggestion
```

Explaining the components is done above in the Components section. If those are not clear enough it is best to improve that section rather than repeating the same message here.

> ## Ribasim core

Reviewer: ```suggestion
# Ribasim core
```

Ribasim is typically used as a command-line interface (CLI). It is distributed as a `.zip`
archive, that must be downloaded and unpacked. It can be placed anywhere, however it is
important that the contents of the zip file are kept together in a directory. The Ribasim
executable is in the main folder.

To download the Ribasim core, download the appropriate zip file for your operating system:

- Ribasim executable - Windows: [ribasim_windows.zip](https://github.com/Deltares/Ribasim/releases/latest/download/ribasim_windows.zip)
- Ribasim executable - Linux: [ribasim_linux.zip](https://github.com/Deltares/Ribasim/releases/latest/download/ribasim_linux.zip)

Note that we currently only support and provide binaries for Windows and Linux, for the x86_64 architecture.

> To check whether the installation was performed successfully, open a terminal and go to the path where the executable is for example 'C:\Ribasim\ribasim_windows'.

Reviewer: ```suggestion
To check whether the installation was performed successfully, open a terminal and go to the path where the executable is, for example `C:\bin\ribasim\`.
```

I noticed the path was not rendering well.

For more information, try '--help'.'
```

> ## Ribasim Python

Reviewer: ```suggestion
# Ribasim Python
```

One can also use Ribasim Python to build entire models from base data, such that your model
setup is fully reproducible.

The Ribasim Python package is [registered in PyPI](https://pypi.org/project/ribasim/) and [conda-forge](https://prefix.dev/channels/conda-forge/packages/ribasim) and can therefore be installed with [pip](https://docs.python.org/3/installing/index.html), [conda](https://docs.conda.io/) or [pixi](https://pixi.sh/):
```
pip install ribasim
```

For documentation please see the [examples](/guide/examples.ipynb) and [API reference](/reference/reference).

> ## Ribasim QGIS plugin

Reviewer: ```suggestion
# Ribasim QGIS plugin
```

The Ribasim QGIS plugin requires [QGIS](https://qgis.org/en/site/) 3.34 or higher.

> The Ribasim QGIS plugin is also distributed as a .zip archive and must be downloaded and installed in QGIS.

Reviewer: ```suggestion
The Ribasim QGIS plugin is only distributed as a .zip archive and must be downloaded and installed in QGIS.
```

> ### Install Ribasim plugin

Reviewer: ```suggestion
## Install Ribasim plugin
```

Firstly, download `ribasim_qgis.zip`:

- QGIS plugin: [ribasim_qgis.zip](https://github.com/Deltares/Ribasim/releases/latest/download/ribasim_qgis.zip).

> In QGIS plugin, go to Plugins menu > Manage and Install Plugins...

Reviewer: ```suggestion
In QGIS, go to Plugins menu > Manage and Install Plugins...
```

![](https://user-images.githubusercontent.com/4471859/224939101-228e068a-875b-4df2-98bb-6ee6a3830ddd.png){fig-align="left"}

> ### Install iMOD plugin

Reviewer: ```suggestion
## Install iMOD plugin
```
